pkg/apis/operator/v1alpha1/elevator_types.go

```diff
@@ -50,6 +52,7 @@ type ElevatorPhaseType string
 // Values of ElevatorPhaseType.
 const (
 	ElevatorPhaseReady       ElevatorPhaseType = "Ready"
+	ElevatorPhasePaused      ElevatorPhaseType = "Paused"
```

Comment:
when is the high time for autogenerating this stuff? we are doing it manually every time, both in kubecarrier, and in bulward.

Reply:
let's add separate issue if we don't have one already and finally do it( just do it :))